src/Integrations/Mailer/SettingsMailer.php

```diff
@@ -214,157 +226,181 @@ public function getSettingsData(string $formId): array
+								],
 								[
 									'component' => 'divider',
 									'dividerExtraVSpacing' => 'true',
 								],
 								[
 									'component' => 'input',
-									'inputName' => $this->getSettingsName(self::SETTINGS_MAILER_SENDER_SUBJECT_KEY),
+									'inputName' => $this->getSettingsName(self::SETTINGS_MAILER_SUBJECT_KEY),
 									'inputFieldLabel' => \__('Subject', 'eightshift-forms'),
 									// translators: %s will be replaced with forms field name.
-									'inputFieldHelp' => \sprintf(\__('Use template tags to use submitted form data (e.g. <code>{field-name}</code>)', 'eightshift-forms'), $formNames),
+									'inputFieldHelp' => \sprintf(\__('
+										Specify email subject with this field.<br /><br />
+										%s', 'eightshift-forms'), $this->getFieldTagsOutput($fieldNameTags)),
 									'inputType' => 'text',
 									'inputIsRequired' => true,
-									'inputValue' => $this->getSettingsValue(self::SETTINGS_MAILER_SENDER_SUBJECT_KEY, $formId),
+									'inputValue' => $this->getSettingsValue(self::SETTINGS_MAILER_SUBJECT_KEY, $formId),
 								],
 								[
 									'component' => 'divider',
 									'dividerExtraVSpacing' => 'true',
 								],
 								[
 									'component' => 'textarea',
-									'textareaName' => $this->getSettingsName(self::SETTINGS_MAILER_SENDER_TEMPLATE_KEY),
-									'textareaFieldLabel' => \__('E-mail content', 'eightshift-forms'),
+									'textareaName' => $this->getSettingsName(self::SETTINGS_MAILER_TEMPLATE_KEY),
+									'textareaFieldLabel' => \__('Content', 'eightshift-forms'),
 									// translators: %s will be replaced with forms field name.
 									'textareaFieldHelp' => \sprintf(\__('
-									Use template tags to use submitted form data (e.g. <code>{field-name}</code>)
-									<details class="is-filter-applied">
-										<summary>Available tags</summary>
-										<ul>
-											%1$s
-										</ul>
-
-										<br />
-										Tag missing? Make sure its field has a <b>Name</b> set!
-									</details>', 'eightshift-forms'), $formNames),
+										Specify email body template with this field. You can use plain text of simple HTML tags.<br /><br />
+										%1$s %2$s', 'eightshift-forms'), $this->getFieldTagsOutput($fieldNameTags), $this->getResponseTagsOutput($formResponseTags)),
 									'textareaIsRequired' => true,
-									'textareaValue' => $this->getSettingsValue(self::SETTINGS_MAILER_SENDER_TEMPLATE_KEY, $formId),
+									'textareaValue' => $this->getSettingsValue(self::SETTINGS_MAILER_TEMPLATE_KEY, $formId),
 								],
-							] : []),
+							]
 						],
-					],
+						[
+							'component' => 'tab',
+							'tabLabel' => \__('Confirmation e-mail', 'eightshift-forms'),
+							'tabContent' => [
+								[
+									'component' => 'intro',
+									'introSubtitle' => \__('Provide the best UX experience for you users by creating a confirmation e-mail like a "thank you" message.', 'eightshift-forms'),
```

Comment:
```suggestion
									'introSubtitle' => \__('Send your users a "thank you" message after they submit the form.', 'eightshift-forms'),
```
UX = user experience 😛